src/infrastructure/directories/api.js

```diff
@@ -252,6 +252,27 @@ const deactivateInvite = async (id, reason, correlationId) => {
   }
 };
 
+const reactivateInvite = async (id, reason, correlationId) => {
+  try {
+    const token = await jwtStrategy(config.directories.service).getBearerToken();
+    await rp({
+      method: 'PATCH',
+      uri: `${config.directories.service.url}/invitations/${id.replace('inv-', '')}`,
+      headers: {
+        authorization: `bearer ${token}`,
+        'x-correlation-id': correlationId,
+      },
+      body: {
+        reason: reason,
+        deactivated: true,
```

Comment:
Think this needs to be deactivated: false

Reply:
Missed the commit... have updated and re-submitted